index.html

```diff
@@ -0,0 +1,114 @@
+          <li><a href="#">English</a></li>
+          <li><a href="#">My Page</a></li>
+          <li><a href="#">Log out</a></li>
+        </ul>
+      </div>
+      <nav class="lower-nav">
+        <div class="title">
+         <a href="index.html"><h2 class="logo-h1">ICS<span class="logo-span">DA</h2></span></a> 
+        </div>
+        <ul class="lower-nav-ul">
+          <li><a href="about.html">About</a></li>
+          <li><a href="#">Program</a></li>
+          <li><a href="#">Join</a></li>
+          <li><a href="#">Sponsor</a></li>
+          <li class="campaign-cdn"><a href="#">Campaign</a></li>
+        </ul>
+        <button class="nav-toggle">
+            <i id="open" class="fa-solid fa-bars"></i>
+            <i id="close" class="fa-solid fa-xmark"></i>
+        </button>
+      </nav>
+    </header>
+    <section class="hero-section">
+            <div class="hero-desc">    
+                <h2 class="hero-title">'Hello Data Science Enthusiasts'</h1>
+               <h1 class="hero-subtitle">International Conference On Statistics and Data Science</h1>
+               <div class="hero-para">
+                   <p>A joyful celebration believing in the value of openness and sharing, creating a positive change with people from over 80 countries is taking place in October, in Okap. </p>
+               </div>
+               <h2 class="date">2023.10.19(THU) ~20(FRI)</h2>
+               <h2 class="location">@Citadelle Laferrière in Okap, Palais Sans Souci and more</h2>
+               </div>
+    </section>
+    <!-- PROGRAMS SECTION -->
+    <section class="programs">
+      <h3 class="program-title">
+          Main Program
+      </h3>
+   
+      <div class="card-container">
+        <div class="cards">
+          <img src="images/lecture.png" alt="lecture">
+          <h4>Lecture</h4>
+          <p>Listen to the speakers from various countries about the messeges of sharing and opening.</p>
+        </div>
+        <div class="cards">
+          <img src="images/exhibition.png" alt="lecture">
+          <h4>CC Exhibition</h4>
+          <p>Listen to the speakers from various countries about the messeges of sharing and opening.</p>
+        </div>
+        <div class="cards">
+          <img src="images/forum.png" alt="lecture">
+          <h4>Forum</h4>
+          <p>Listen to the speakers from various countries about the messeges of sharing and opening.</p>
+        </div>
+        <div class="cards">
+          <img src="images/workshop.png" alt="lecture">
+          <h4>workshop</h4>
+          <p>Listen to the speakers from various countries about the messeges of sharing and opening.</p>
+        </div>
+        <div class="cards">
+          <img src="images/ignite.png" alt="lecture">
+          <h4>CC Ignite</h4>
+          <p>Listen to the speakers from various countries about the messeges of sharing and opening.</p>
+        </div>
+      </div>
+      <button class="card-btn">Join ICSDA</button>
+    </section>
```

Comment:
- [ ] According to the guidelines, your footer  contains a logo aligned to the left and text aligned to the right. please fix that
- [ ] kindly reduce the height of the footer


![Screenshot (461)_LI](https://user-images.githubusercontent.com/65298048/191360364-431dd140-5ed7-41b3-bb3d-2ccf59c953a2.jpg)